pom.xml

```diff
@@ -100,16 +100,28 @@
         </exclusion>
       </exclusions>
     </dependency>
-
     <!-- Needed for audit functionality -->
-    <!--
     <dependency>
```

Comment:
Nevermind, I think I get it.  The dependency is included, but not 'activated' by default, right?